packages/accordions/src/styled/accordion/StyledButton.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export const COMPONENT_ID = 'accordions.button';
+
+export interface IStyledButton {
+  isCompact?: boolean;
+  isHovered?: boolean;
+}
+
+/**
+ * 1. Remove dotted outline from Firefox on focus.
+ */
+export const StyledButton = styled.button.attrs<IStyledButton>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledButton>`
+  outline: none;
+  border: none;
+  background: transparent;
+  padding: ${props =>
+    props.isCompact
+      ? `${props.theme.space.base * 1.5}px ${props.theme.space.base * 3}px`
```

Comment:
We would want this to follow base 4 spacing. 4px or 8px

Reply:
Sounds good. I can make these adjustments once the artboards are updated accordingly 👍.

@rossmoody @steue

Reply:
@rossmoody i remember this conversation. did we end up changing the `isCompact` padding on the rest of our components to base-4? currently i still see them at `6px` and `12px` for vertical and horizontal padding respectively.

to clarify, i'm completely onboard on updating this to base-4, i just want some clarification on the whole system's perspective on this spacing so the accordions can match.

Reply:
Yeah, there are a couple outstanding tickets in queue to update `Dropdowns` and `Forms` components to base-4 spacing. `Menus` spacing just recently merged to follow. I wouldn't say you have to follow their spacing increments, just that we as a system adhere to base-4.

Reply:
Awesome. I'll make those adjustments. @hzhu

Reply:
Sounds good, the vertical padding has been updated.